src/main/java/org/medibloc/panacea/BaseWallet.java

```diff
@@ -14,7 +12,7 @@
 import java.util.List;
 
 @ToString
-class BaseWallet {
+public class BaseWallet {
```

Comment:
I'd like to ask if this `BaseWallet` class needs to be public.

Reply:
@youngjoon-lee 
`public` keyword was added automatically.
I will removed now.